src/PortingAssistant.Client/Program.cs

```diff
@@ -136,7 +144,10 @@ public static void Main(string[] args)
 
                         TraceEvent.Start(Log.Logger, $"Applying porting actions to projects in {cli.SolutionPath}");
                         var portingResults = portingAssistantClient.ApplyPortingChanges(portingRequest);
-                        reportExporter.GenerateJsonReport(portingResults, cli.SolutionPath, cli.OutputPath);
+                        if (!cli.DisabledMetrics)
```

Comment:
confirm if only metrics need to be disabled or the file generation itself

Reply:
I've discussed this with @sunwoo-ms offline. We are starting with file generation since we have seen OOM exceptions there and is a quick fix. Disabling metrics altogether will be a more involved process and will be a next step or completed as needed.